views/uikit_layout.html

```diff
@@ -25,13 +25,24 @@
     <link rel="stylesheet" href="{{=URL('static','uikit-3/css/uikit.min.css')}}" />
     <script src="{{=URL('static','uikit-3/js/uikit.min.js')}}"></script>
     <script src="{{=URL('static','uikit-3/js/uikit-icons.min.js')}}"></script>
+    <script src="{{=URL('static','js/web2py.js')}}"></script>
```

Comment:
Could we just include webpy_ajax.html on pages that really need it?

At best this downloads a useless copy of jquery for every page, at worst introduces web2py behaviours which then fight with our JS (the "loading..." message appearing on all buttons I've seen in the past).

Reply:
Ah right. I didn't realise that was the reason we had killed it. I guess we might bung it in on pages that use Ajax then?